Feature: ClamScan container config and scan file.

sfaulkner89 opened the pull request.

GrahamMWillis commented on packages/web-service/src/services/virus-scan.js:
```diff
@@ -0,0 +1,38 @@
+export async function scanFile (filename) {
+  return ClamScan.then(async (clamscan) => {
+    try {
+      const dir = `../..${process.env.SCANDIR}/${filename}`
```
This needs to be an absolute path I think

GrahamMWillis commented on docker/cloud-services.yml:
```diff
@@ -68,3 +68,25 @@ services:
+
+volumes:
+  virus-db:
+  scandir:
```
Not a big deal but the convention is a CR at the EOF

sfaulkner89 commented:
After talking to Kealan we decided that if the scan fails, the file should be deleted on my end for consistency's sake. I've added that in, fixed your comments and the code smells.

GrahamMWillis commented on packages/web-service/src/services/virus-scan.js:
```diff
@@ -0,0 +1,45 @@
+const ClamScan = new NodeClam().init(options)
+
+export async function scanFile (filename) {
+  const dir = `.${process.env.SCANDIR}/${filename}`
```
I still think this should be an absolute path - do you want to have a chat about it?
